app/playground/src/PlaygroundList.js

```diff
@@ -5,27 +5,29 @@ import { FlatList, View } from 'react-native';
 import type { NavigationType } from '@kiwicom/mobile-navigation';
 import { WithStorage } from '@kiwicom/mobile-shared';
 
-import '../../shared/src/forms/datePicker/__tests__/DatePicker.test.js';
-import '../../hotels/src/allHotels/__tests__/LoadMoreButton.test.js';
-import '../../hotels/src/allHotels/searchForm/locationPicker/__tests__/RecentSearches.test.js';
-import '../../hotels/src/allHotels/searchForm/locationPicker/__tests__/SuggestionList.test.js';
-import '../../hotels/src/filter/stars/__tests__/StarsFilter.test.js';
-import '../../hotels/src/filter/stars/__tests__/StarsPopup.test.js';
-import '../../hotels/src/gallery/__tests__/PhotosStripeHeader.test.js';
-import '../../hotels/src/singleHotel/bookNow/__tests__/BookNow.test.js';
-import '../../hotels/src/singleHotel/roomList/__tests__/RoomBadges.test.js';
-import '../../hotels/src/singleHotel/roomList/__tests__/RoomDescription.test.js';
-import '../../hotels/src/singleHotel/roomPicker/__tests__/RoomPicker.test.js';
-import '../../shared/src/__tests__/ReadMore.test.js';
-import '../../shared/src/__tests__/Text.test.js';
-import '../../shared/src/badge/__tests__/AdaptableBadge.test.js';
-import '../../shared/src/buttons/__tests__/Button.test.js';
-import '../../shared/src/buttons/__tests__/IncrementDecrementButtons.test.js';
-import '../../shared/src/cards/__tests__/SimpleCard.test.js';
-import '../../shared/src/forms/__tests__/Slider.test.js';
-import '../../shared/src/popup/__tests__/ButtonPopupPlayground.test.js';
 import PlaygroundListItem from './PlaygroundListItem';
 import PlaygroundRenderer from './PlaygroundRenderer';
+// PlaygroundImportsStart
```

Comment:
@tbergq What about creating one file only with these autogenerated imports and include it here?  I don't like this semi-automatic tweaking of one part of our code...

Reply:
Yeah, that sounds like a better idea. 👍